src/modules/refresh.js

```diff
@@ -0,0 +1,15 @@
+import { getScores, display } from './api.js';
+
+export const refresh = async () => {
+  const displ = document.getElementById('displ');
+  displ.innerHTML = '';
+  const games = await getScores();
+  games.forEach((game) => {
+    display(game);
+  });
+};
+
+export const refreshEventHandler = (e) => {
+  e.preventDefault();
+  refresh();
+};
```

Comment:
- [ ] Hi @jugosack, great effort working on your leaderboard API, 👍 According to the project requirement, when a particular score is submitted by a user it is posted to the API and it can reflect on the leaderboard when the `refresh` button is clicked. But currently, in your project, the score submitted by a user is immediately populated to the board before the `refresh` button is clicked. Kindly work on this and make the leaderboard function in this manner.

   >N.B: 1. The submit button should hit the POST API endpoint and post user entries.
   >2. When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change.

Reply:
> 

@brainconnect93 Thanks for your remark. I will improve the project.